drivers/wdm/audio/backpln/portcls/irp.cpp

```diff
@@ -9,13 +9,8 @@
  *                  27 Jan 07   Created
  */
 
-#include "private.hpp"
-
-#ifndef YDEBUG
-#define NDEBUG
-#endif
 
-#include <debug.h>
+#include "private.hpp"
```

Comment:
strip one newline above #include "private.hpp" as it will help git compare to better align what should be aligned.

Reply:
Fixed.